### DISCUSSION TITLE
Verhalten von SAE, wenn Zähler zurückgesetzt wird

### DISCUSSION THREAD

**Discussion by zapccu** [camueller/SmartApplianceEnabler#585] [Allgemein]
Hallo,
leider liefert meine Gastherme (Viessmann) nur Zeitraum bezogene Leistungszähler, also Leistung für den aktuellen Tag, die aktuelle Woche usw.
Ich verwende aktuell den Zähler Leistung pro aktuellem Jahr. Der wird jedoch am Ende des Jahre auf 0 zurückgesetzt. Dadurch wird die Differenz zwischen aktuellem und vorherigen Wert genau einmal negativ. Wie geht SAE damit um? Welcher Wert wird in diesem Fall für SMA bereitgestellt?

**Comment by camueller** [comment]
Der SAE meldet immer die aktuelle Leistungsaufnahme an den Sunny Home Manager und dieser an das Sunny Portal. Letztres macht dann alle Summierungen. Wenn Deine Gastherme die aktuelle Leistungsaufnahme nicht bereitstellt, kannst Du ja einen eigenen Zähler vor diese schalten und vom SAE abfragen lassen.
